contrib/sip_proxy/filters/network/source/router/router_impl.cc

```diff
@@ -168,11 +233,9 @@ FilterStatus Router::transportBegin(MessageMetadataSharedPtr metadata) {
   metadata_ = metadata;
   route_ = callbacks_->route();
   if (!route_) {
-    ENVOY_STREAM_LOG(debug, "no route match domain {}", *callbacks_, metadata->domain().value());
+    ENVOY_STREAM_LOG(debug, "no route matched", *callbacks_);
     stats_.route_missing_.inc();
-    callbacks_->sendLocalReply(AppException(AppExceptionType::UnknownMethod, "no route for method"),
-                               true);
-    std::cout << "DDD ---------------1\n";
+    throw AppException(AppExceptionType::UnknownMethod, "envoy no match route found");
     return FilterStatus::StopIteration;
```

Comment:
This line doesn't seem to be reachable now. Can be dropped.

Reply:
removed.